components/ui/screen-loader.tsx

```
@@ -1,12 +1,12 @@
-import { IconLoader2 } from '@tabler/icons-react';
+import { LoaderCircle } from 'lucide-react';
 import type { FC } from 'react';
 
 type ScreenLoaderProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)